diff -r 0d4d25ef06e1 browser/branding/720brs/brand.dtd
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/browser/branding/720brs/brand.dtd	Wed Sep 12 15:49:14 2012 +0800
@@ -0,0 +1,4 @@
+<!ENTITY  brandShortName        "720浏览器">
+<!ENTITY  brandFullName         "720浏览器">
+<!ENTITY  vendorShortName       "720browser.com">
+<!ENTITY  trademarkInfo.part1   " ">
diff -r 0d4d25ef06e1 browser/branding/720brs/brand.properties
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/browser/branding/720brs/brand.properties	Wed Sep 12 15:49:14 2012 +0800
@@ -0,0 +1,10 @@
+brandShortName=720浏览器
+brandFullName=720浏览器
+vendorShortName=720browser.com
+syncBrandShortName=同步
+
+homePageSingleStartMain=720浏览器起始页，内建的快速搜索主页
+homePageImport=从 %S 导入您的主页
+
+homePageMigrationPageTitle=主页选择
+homePageMigrationDescription=请选择您想要使用的主页:
diff -r 0d4d25ef06e1 browser/chrome/browser/browser.dtd
--- a/browser/chrome/browser/browser.dtd	Fri Jun 22 07:27:26 2012 +0800
+++ b/browser/chrome/browser/browser.dtd	Wed Sep 12 15:49:14 2012 +0800
@@ -653,3 +653,8 @@
 <!-- LOCALIZATION NOTE (markupButton.accesskey): The key bound to the Markup panel's
 toolbar button -->
 <!ENTITY markupButton.accesskey          "M">
+
+<!ENTITY responsiveDesignTool.accesskey "R">
+<!ENTITY responsiveDesignTool.commandkey "M">
+
+<!ENTITY responsiveDesignTool.label   "Responsive Design View">
diff -r 0d4d25ef06e1 dom/chrome/dom/dom.properties
--- a/dom/chrome/dom/dom.properties	Fri Jun 22 07:27:26 2012 +0800
+++ b/dom/chrome/dom/dom.properties	Wed Sep 12 15:49:14 2012 +0800
@@ -126,3 +126,7 @@
 # LOCALIZATION NOTE: Do not translate "Blob", "mozSlice", or "slice"
 MozSliceWarning=对 Blob 使用 mozSlice 已过时，请使用 slice 替代。
 OnuploadprogressWarning=XMLHttpRequest 的 onuploadprogress 属性已过时。
+
+# LOCALIZATION NOTE: Do not translate "java" or "Packages".
+JavaPackagesWarning=Use of the 'java' and 'Packages' DOM objects to access Java functionality is deprecated. Support will be removed soon.
+diff -r -pU0 720brs/chrome/zh-CN/locale/zh-CN/mozapps/plugins/plugins.dtd firefox/chrome/zh-CN/locale/zh-CN/mozapps/plugins/plugins.dtd
diff -r 0d4d25ef06e1 toolkit/chrome/mozapps/plugins/plugins.dtd
--- a/toolkit/chrome/mozapps/plugins/plugins.dtd	Fri Jun 22 07:27:26 2012 +0800
+++ b/toolkit/chrome/mozapps/plugins/plugins.dtd	Wed Sep 12 15:49:14 2012 +0800
@@ -43,3 +43,13 @@
 <!ENTITY report.disabled                                     "禁用崩溃报告。">
 <!ENTITY report.failed                                       "提交失败。">
 <!ENTITY report.unavailable                                  "无可用报告。">
+
+<!-- LOCALIZATION NOTE (unsupportedPlatform.learnMore): Mobile only. This text is used to link to a SUMO page explaining why Flash is not
+     supported on this device. Use the unicode ellipsis char, \u2026, or use "..." if \u2026 doesn't suit traditions in your locale. -->
+<!ENTITY unsupportedPlatform.learnMore                       "Learn More…">
+<!-- LOCALIZATION NOTE (unsupportedPlatform.post): Mobile only. Include text here if needed for your locale. -->
+<!ENTITY unsupportedPlatform.post                            "">
+
+<!-- LOCALIZATION NOTE (unsupportedPlatform.pre): Mobile only. Flash (the only plugin available on mobile)
+     is not supported on some devices. Include a trailing space as needed. -->
+<!ENTITY unsupportedPlatform.pre                             "We're very sorry, but &brandShortName; can't play Flash on this device. ">
